tests/testfont.c

```diff
@@ -0,0 +1,55 @@
+#ifdef WIN32
```

Comment:
file should be called `testfonts.c` to be consistent with what you added in Makefile.am